pkg/logging/filelogging.go

```diff
@@ -0,0 +1,56 @@
+package logging
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/openshift/installer/pkg/asset"
+)
+
+// LogCreatedFiles checks all the asset files created and logs it for the user to see.
+func LogCreatedFiles(cmdName string, directory string, targets []asset.WritableAsset) {
+	directory = strings.TrimRight(directory, `/`)
```

Comment:
https://golang.org/pkg/path/filepath/ use this package instead of creating these one of trims or splits.